org.eclipse.winery.frontends/app/tosca-management/src/app/wineryLicenseModule/wineryLicense.component.html

```diff
@@ -11,50 +11,174 @@
+                                              *ngIf="!isFailed()"></mat-progress-bar>
+                            <mat-progress-bar mode="determinate" value="50" color="warn"
+                                              *ngIf="isFailed()"></mat-progress-bar>
+                            <div style="float: right">Status: {{getStatus()}}</div>
+                            <br>
+                        </div>
+                        <div>
+                            <button class="btn btn-primary" (click)="onSubmitTemplateBased()"
+                                    [disabled]="loadingbar || !firstFormGroup.valid"> Next
+                            </button>
+                            <button style="margin-left: 12px" class="btn btn-primary" (click)="reset()"
+                                    *ngIf="isFailed()">
+                                Reset
+                            </button>
+                        </div>
+                    </form>
+                </mat-step>
+                <mat-step completed="false">
+                    <form [formGroup]="secondFormGroup">
+                        <ng-template matStepLabel>Check Compatibility</ng-template>
+                        <br>
+                        <mat-label style="font-size: 15px">{{foundLicenses}}</mat-label>
+                        <br><br>
+                        <div>
+                            <button class="btn btn-primary" matStepperPrevious>Back</button>
+                            <button style="margin-left: 12px" class="btn btn-primary"
+                                    (click)="onSubmitCheckCompatibility()">Next
+                            </button>
+                        </div>
+                    </form>
+                </mat-step>
+                <mat-step>
+                    <ng-template matStepLabel>Select License</ng-template>
+                    <div>
+                        <mat-action-list>
+                            <mat-list-item *ngFor="let compatibleLicense of compatibleLicenses">
+                                <button mat-list-item
+                                        (click)="selectLicense(compatibleLicense)"> {{compatibleLicense}} </button>
+                            </mat-list-item>
+                        </mat-action-list>
+                        <button class="btn btn-primary" (click)="stepper.reset()">Reset</button>
+                    </div>
+                </mat-step>
+            </mat-horizontal-stepper>
+        </div>
+
+        <ng-template #confirmSaveModal>
+            <winery-modal-header [modalRef]="confirmSaveModalRef"
+                                 [title]="'Are you sure you want to save the license text without modifying it?'">
+            </winery-modal-header>
+            <winery-modal-body>
+                <p>
+                    Depending on the selected license, some fields like the date or the copyright holders have to be
```

Comment:
Please use `some fields, such as the date or copyright holders, must be set!`.

[like vs. such as](http://www.differencebetween.net/language/grammar-language/difference-between-such-as-and-like/#:~:text=%E2%80%9CLike%E2%80%9D%20is%20used%20when%20comparing%20persons%20or%20things%20and%20describing,objects%20of%20comparison%20are%20definite.)
[must vs. have to](https://www.gymglish.com/en/gymglish/english-grammar/must-vs-have-to#:~:text=Have%20to%20mainly%20expresses%20general,and%20need%20are%20not%20used.)

Reply:
changed